packages/bot-skeleton/src/services/api/ticks_service.js

```diff
@@ -156,7 +158,7 @@ export default class TicksService {
             ...(tickSubscription || []),
         ];
 
-        Promise.all(subscription.map(id => doUntilDone(() => this.api.unsubscribeByID(id))));
+        Promise.all(subscription.map(id => doUntilDone(() => this.api.forgetStream(id))));
```

Comment:
Do we need `forgetStream`? Why not just use forget? 
```suggestion
        Promise.all(subscription.map(id => doUntilDone(() => this.api.forget(id))));
```

Reply:
> Do we need `forgetStream`? Why not just use forget?

Updated